eventmesh-connectors/eventmesh-connector-rocketmq/build.gradle

```diff
@@ -16,25 +16,33 @@
  */
 
 List rocketmq = [
+        "org.apache.rocketmq:rocketmq-acl:$rocketmq_version",
         "org.apache.rocketmq:rocketmq-client:$rocketmq_version",
-        "org.apache.rocketmq:rocketmq-broker:$rocketmq_version",
         "org.apache.rocketmq:rocketmq-common:$rocketmq_version",
-        "org.apache.rocketmq:rocketmq-store:$rocketmq_version",
-        "org.apache.rocketmq:rocketmq-namesrv:$rocketmq_version",
-        "org.apache.rocketmq:rocketmq-tools:$rocketmq_version",
-        "org.apache.rocketmq:rocketmq-remoting:$rocketmq_version",
-        "org.apache.rocketmq:rocketmq-logging:$rocketmq_version",
-        "org.apache.rocketmq:rocketmq-srvutil:$rocketmq_version",
         "org.apache.rocketmq:rocketmq-filter:$rocketmq_version",
-        "org.apache.rocketmq:rocketmq-acl:$rocketmq_version",
+        "org.apache.rocketmq:rocketmq-logging:$rocketmq_version",
+        "org.apache.rocketmq:rocketmq-remoting:$rocketmq_version",
         "org.apache.rocketmq:rocketmq-srvutil:$rocketmq_version",
-
+        "org.apache.rocketmq:rocketmq-store:$rocketmq_version",
+        "org.apache.rocketmq:rocketmq-srvutil:$rocketmq_version"
 ]
 
 dependencies {
     api project(":eventmesh-openconnect:eventmesh-openconnect-java")
     implementation project(":eventmesh-common")
     implementation rocketmq
+    implementation("org.apache.rocketmq:rocketmq-broker:$rocketmq_version") {
+        // Remove logging backend implementations
+        exclude group: 'ch.qos.logback', module: 'logback-classic'
+    }
```

Comment:
I chose this way by observing the generated POM and `*.module` files that are published to Maven Central.

The `configurations` solution you propose gives a POM file like [`eventmesh-storage-rocketmq-1.10.0-release.pom`](https://repo1.maven.org/maven2/org/apache/eventmesh/eventmesh-storage-rocketmq/1.10.0-release/eventmesh-storage-rocketmq-1.10.0-release.pom) and a module file like [`eventmesh-storage-rocketmq-1.10.0-release.module`](https://repo1.maven.org/maven2/org/apache/eventmesh/eventmesh-storage-rocketmq/1.10.0-release/eventmesh-storage-rocketmq-1.10.0-release.module). As you can see, `logback-classic` is excluded from **each** dependency.

By adding the exclusion to `rocketmq-broker`, only `rocketmq-broker` has it, while other dependencies have no exclusions.

**Remark**: the problem with RocketMQ will be solved, when apache/rocketmq#5347 is fixed, although nothing indicates it will ever be solved. The newest version of `rocketmq-broker` replaces the dependency on Logback with a dependency on:

```xml
<dependency>
    <groupId>io.github.aliyunmq</groupId>
    <artifactId>rocketmq-shaded-slf4j-api-bridge</artifactId>
    <version>1.0.0</version>
</dependency>
```

that will need to be excluded as well.

Do you really need `rocketmq-broker` and the other dependencies? Looking at the code, all you need it `rocketmq-tools`.

Reply:
>although nothing indicates it will ever be solved

🤣

>As you can see, `logback-classic` is excluded from **each** dependency.

Gradle isn't Maven after all, and while the pom file generated by the `configurations` block isn't accurate from a Maven perspective, it does simplify configuration when using Dependency Lists in Gradle.

So I think we'll just use the `configurations` block and keep the List intact in the build.gradle that uses the `List`. Other build.gradles that don't use the `List` keep the surgical strikes.

>Do you really need `rocketmq-broker` and the other dependencies?

There really is no reference to the `rocketmq-broker` package in the code. I'll create an issue for this, and there will be no need to make changes to this issue in this PR for now.

Reply:
I restored the list and put a comment with the link to the RocketMQ issue that will allow you to remove the exclusion.